modules/core/src/main/java/org/apache/fluo/core/impl/FluoConfigurationImpl.java

```diff
@@ -102,6 +104,51 @@ public static int getTxCommitMemory(FluoConfiguration conf) {
     return m;
   }
 
+  public static final String TX_INFO_CACHE_SIZE = FLUO_IMPL_PREFIX + ".tx.failed.cache.size.mb";
+  public static final long TX_INFO_CACHE_SIZE_DEFAULT = 10000000;
+
+  /** 
+   * Gets the value of the property {@value #TX_INFO_CACHE_SIZE} if set,
+   * else gets the default value {@value  #TX_INFO_CACHE_SIZE_DEFAULT}
+   * 
+   * @param conf The FluoConfiguration.
+   * @return The size of the cache value from the property value {@value #TX_INFO_CACHE_SIZE}
+   *     if it is set, else the value of the default value {@value #TX_INFO_CACHE_SIZE_DEFAULT}
+   *     
+   * @since 1.2.0
+   */
+
+  public static long getTxInfoCacheSize(FluoConfiguration conf) {
+    long size = conf.getLong(TX_INFO_CACHE_SIZE, TX_INFO_CACHE_SIZE_DEFAULT);
+    if (size <= 0) {
+      throw new IllegalArgumentException("Cache size must be positive for " + TX_INFO_CACHE_SIZE);
+    }
+    return size;
+  }
+
+  public static final String TX_INFO_CACHE_TIMEOUT =
+      FLUO_IMPL_PREFIX + ".tx.failed.cache.expireTime.minutes";
```

Comment:
I think minutes makes sense for this property.  However I am thinking we should change the unit to milliseconds for consistency.  Some other time based configs may need more granularity.  Going forward, I think it would be nice if all new time based configs added had the same unit.  This results in a less confusing experience when considering the config as a whole.

We can use a suffix of `.ms` for the property name.